app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
```

Comment:
Whats up with these files?

Reply:
Has been "fixed" , by changing all our devise_error_messages! to resource.errors.any?

Is it the correct path? 

There were many google/stackoverflow threads mentioning thousand solutions...

Reply:
Fixed, deleted the file ( I have no idea how that got in my drive... )